Makefile

```diff
@@ -16,3 +16,7 @@ endif
 ifeq ($(OSTYPE),OpenBSD)
 	include make-freebsd.mk
 endif
+
+ifeq ($(OSTYPE),NetBSD)
+	include make-netbsd.mk
```

Comment:
Is it possible to unify `make-freebsd.mk` and `make-netbsd.mk` into `make-bsd.mk`?